doc/publishing-api-syntactic-usage.md

```diff
@@ -137,6 +137,14 @@ TODO: Request/Response detail
 Note that link sets can be created before or after the PUT requests for the content item.
 No downstream requests will be sent if the content item doesn't exist yet.
 
+To remove a `link_type` from the links, update it with an empty array. For example:
```

Comment:
I think I would phrase this as 
"To delete all the links of a `link_type`, update it with an empty array. This will also remove the reference to the `link_type` from the links"